.ci/pipeline_definitions

```diff
@@ -12,7 +10,6 @@ gardener-extension-runtime-gvisor:
         - linux/arm64
         dockerimages:
           gardener-extension-runtime-gvisor:
-            registry: 'gcr-readwrite'
             image: 'eu.gcr.io/gardener-project/gardener/extensions/runtime-gvisor'
```

Comment:
Shouldn't this be updated to point to `europe-docker.pkg.dev/gardener-project/releases`?

Reply:
oh dear. It seems I missed the second commit on that pull-request :-/

Reply:
pushed the second commit as well (I already had it in my local repository) :-)